addons/frag/functions/fnc_dev_trackHitBox.sqf

```diff
@@ -0,0 +1,75 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a hit box outline to an object, outdated for unit hits as they
+ * use hit-point locations.
+ *
+ * Arguments:
+ * 0: Object that should have it's hit box drawn. <OBJECT>
+ * 1: Add sphere at object origin. <BOOL> (Default: true)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_frag_fnc_dev_trackHitBox;
+ *
+ * Public: No
+ */
+
+params [
+    ["_object", objNull, [objNull]],
+    ["_addSphere", true, [true]]
+];
+TRACE_2("Adding hitbox",_object,_addSphere);
+
+if (isNull _object) exitWith {};
```

Comment:
As a developement tool, I think it's more helpful to have it catch cases when something else breaks and it is null rather than for it to go under the radar. And on that noteI'll add it to `fnc_dev_trackObj` as awell.

Reply:
done by https://github.com/acemod/ACE3/pull/9728/commits/66c566b772c9ff7f7a66a77e86f909b0fcab9f67